src/site/includes/operatingStatusFlags.drupal.liquid

```diff
@@ -0,0 +1,68 @@
+<style>
+  .operating-status-flag {
+    border-radius: 5px;
+  }
+
+  a.operating-status-link {
+    color: initial;
+    text-decoration: none;
+  }
+
+  .operating-status-flag .fa-chevron-right {
+    font-size: 12px;
+  }
+
+  .operating-status-flag-notice:hover {
+    background: rgba(155, 218, 241, .4);
+  }
+
+  .operating-status-flag-warning:hover {
+    background: rgba(250, 217, 128, .4);
+  }
+
+  .operating-status-flag-error:hover {
+    background: rgba(229, 147, 147, .4)
+  }
+
+  .operating-status-flag-notice:active {
+    background: rgba(155, 218, 241, 1);
+  }
+
+  .operating-status-flag-warning:active {
+    background: rgba(250, 217, 128, 1);
+  }
+
+  .operating-status-flag-error:active {
+    background: rgba(229, 147, 147, 1)
+  }
+</style>
```

Comment:
This all seems reasonable to me, but wanted to make sure this didn't throw any red flags (:bad-pun-dog:) for any of our designers. The thing I'm looking at specifically is making the `a.operating-status-link` look like regular text. Given the context, it seems to make sense, but I'm not a designer for a reason. :sweat_smile: 

cc @CrystabelReiter @1Copenut 

Unfortunately, because these pages are hidden from production, it also looks like they're hidden from our [review instances](https://vetsgov-pr-11155.herokuapp.com/pittsburgh-health-care/). :disappointed:

Reply:
@cvalarida This is the design we have from @rtwell so we should be fine :) 

![image](https://user-images.githubusercontent.com/8558266/69550441-03ebf700-0f69-11ea-8b7a-e2115af4bc0a.png)